Meadowlark-js/packages/meadowlark-core/src/validation/DocumentIdentityExtractor.ts

```diff
@@ -62,7 +72,12 @@ function documentIdentitiesFrom(
   if (isReferenceElement(identityReferenceComponent)) {
     // SchoolYearEnumerations are an API one-off expressed as two levels in the document body
     if (identityReferenceComponent.sourceProperty.type === 'schoolYearEnumeration') {
-      documentPath.push('schoolYearTypeReference');
+      if (identityReferenceComponent.sourceProperty.roleName) {
```

Comment:
I need to jog my memory on the better way to handle checking if there is a value here. Don't tell me yet! I need to look into and get it to sink into my memory.